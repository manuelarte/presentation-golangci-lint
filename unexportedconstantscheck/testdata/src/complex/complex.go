package complex

// rate the case
const rate = 0 // want `unexported constant "rate" should be prefixed with _`

const one, two = "ONE", "TWO" // want `unexported constants "ONE, TWO" should be prefixed with _`
