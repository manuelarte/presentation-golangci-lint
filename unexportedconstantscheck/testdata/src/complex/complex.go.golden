package simple

// rate the case
const _rate = 0 // want `unexported constant "rate" should be prefixed with _`

const _one, _two = "ONE", "TWO" // want `unexported constants "ONE", TWO" should be prefixed with _`
